README.md

```diff
@@ -3,6 +3,17 @@
 ## Data
 See [data readme](data/README.md)
 
+## Add a new image
```

Comment:
I'm thinking we'll want to merge the `data` and `tryon_folder` directories at some point since they contain the same structure?

Reply:
Ah! yes maybe we can rename `data` to just be `m3d-completed` folder, since its helpful for the data directory and it has _everything_ (i.e. not custom data but includes all the masks etc etc)